example/docker-compose/distributed/tempo-distributed.yaml

```diff
@@ -25,14 +25,13 @@ ingester:
   lifecycler:
     ring:
       replication_factor: 3
-
-memberlist:
```

Comment:
Thanks for the fix @mapno. 

I'm wondering if we can add an e2e test here, it will be useful to protect against this in future PRs. Adding an example/docker-compose helps but adds a manual testing step.

Reply:
Added the e2e test!